### PR TITLE
Use IP address as the default hostname (closes #575)

### LINKED ISSUE #575: MS Edge fails to open an url with NetBIOS hostname
Maybe it is just issue with our network only, but Edge just shows "Hmm, we can't reach this page" when starting from command line(through `microsoft-edge://` protocol) if the page url has NetBIOS name as the hostname. I've tried to discuss the issue with M$ a year ago: https://social.technet.microsoft.com/Forums/en-US/06ae1597-6c28-4fd2-95ac-f52df6c72d34/microsoft-edge-doesnt-open-pages-via-microsoftedge-protocol-and-netbios-name-as-a-hostname?forum=WinPreview2014General, but they have said: "It works for us", and that's all. BTW I'm still encountering the issue on my Windows 10 RTM machine, and I know a guy inside our company that has experienced the issue on the Windows 10 with 1511 update (service-pack). Also note that we are using NetBIOS name for the hostname by default, so you must explicitly specify the hostname, if you want to work with MS Edge and `testcafe`.

### PR DESCRIPTION
**Pull request by AndreyBelym**
\cc @inikulin @AlexanderMoskovkin

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -14,7 +15,7 @@ async function getValidHostname (hostname) {
             throw new GeneralError(MESSAGE.invalidHostname, hostname);
     }
     else
-        hostname = await endpointUtils.getMyHostname();
+        hostname = await ip.address();
```
Comment (inikulin) [inline src/index.js]: AFAIK it's sync

**Comment by inikulin** [comment]
\r-

**Comment by inikulin** [comment]
BTW, endpointUtils already use `ip` internally, I'll expose `address` there

**Comment by inikulin** [comment]
@AndreyBelym Just published v1.0.2 with this functionality: https://github.com/inikulin/endpoint-utils#get-ip-address-of-current-machine

**Comment by AlexanderMoskovkin** [comment]
just an unstable qunit test

**Comment by inikulin** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Pull request merged**